xikolo-ios/ViewControllers/News/NewsViewController.swift

```diff
@@ -30,11 +30,6 @@ class NewsViewController : AbstractTabContentViewController {
         syncContent()
     }
 
-    override func updateUIAfterLoginLogoutAction() {
-        super.updateUIAfterLoginLogoutAction()
-        syncContent()
```

Comment:
This line may be important, although I'm not sure why. When I logout on the master, platform events don't get displayed anymore which differs from the behavior we just experienced in your branch. Lets test this again